app/styles/app.scss

```diff
@@ -1385,45 +1384,94 @@ section {
       //margin-top: 40px;
 
       .external-video {
-        height: calc(100vh)!important;
+        height: calc(100vh - 140px) !important;
       }
     }
 
-    .section-heading {
-      position: absolute;
-      padding: 10px;
-      height: 65px;
-      color: white;
-      width: 100%;
-      font-size: 38px;
-      background: #363636;
-      display: flex;
-      padding-left: 10px;
+    &:hover {
+      .section-heading {
+        display: flex;
+      }
+    }
+  }
 
-      p {
-        min-width: 95%;
-        font-size: 20px;
-        margin: 14px 0;
+  .section-heading {
+    position: fixed;
+    top: 94px;
+    padding: 10px 20px;
+    color: white;
+    background: #363636;
+    padding-left: 10px;
+    z-index: 9999;
+    font-size: 18px;
+    transition: .5s;
+
+    &.full-width-header {
+      width: 100% !important;
+      max-width: 100vw;
+      overflow: hidden;
+      transition: .5s;
+    }
+
+    .title-text {
+      padding: 15px;
+      display: inline-block;
+      @media (max-width: 767px) {
+        padding-left: 10px;
+      }
+    }
+
+    .right-section {
+      float: right;
+      margin: 0;
+      padding: 0;
+
+      #goToDisqusLink {
+        margin-right: 10px;
+        vertical-align: super;
+        a {
+          font-size: 18px;
+          color: white;
+          background: darken(#363636, 10%);
+          padding: 5px 10px;
+          border-radius: 12px;
+          cursor: pointer;
+
+          &:hover {
+            text-decoration: underline;
+          }
+        }
       }
       .toggle-button {
         background: transparent;
         color: white;
         border: none;
         cursor: pointer;
-        width: 20px;
-        height: 20px;
+        height: 100%;
+        padding: 0;
+        margin: 0;
+        font-size: 38px;
 
         &:hover {
           color: #d95787;
         }
-
       }
     }
 
-
-    &:hover {
-      .section-heading {
-        display: flex;
```

Comment:
Don't change section heading class, it's designed using FlexBoxes design concepts.